core/src/commonMain/kotlin/com/xebia/functional/xef/llm/openai/models.kt

```diff
@@ -89,6 +97,52 @@ data class ChatCompletionRequestWithFunctions(
   @SerialName("function_call") val functionCall: Map<String, String>,
 )
 
+/*
+"functions": [
+    {
+      "name": "get_current_weather",
+      "description": "Get the current weather in a given location",
+      "parameters": {
+        "type": "object",
+        "properties": {
+          "location": {
+            "type": "string",
+            "description": "The city and state, e.g. San Francisco, CA"
+          },
+          "unit": {
+            "type": "string",
+            "enum": ["celsius", "fahrenheit"]
+          }
+        },
+        "required": ["location"]
+      }
+    }
+  ]
+ */
+@Serializable
+data class CFunction(
```

Comment:
This seems to be duplicated from the file `com.xebia.functional.xef.llm.openai.functions.CFunction`. What is the relation between them?

Reply:
That was a mistake during merging, I removed the duplicated model in `com.xebia.functional.xef.llm.openai.functions.CFunction`. Thank you for catching this @diesalbla!